### DISCUSSION TITLE
CVE-2024-25723

### DISCUSSION THREAD

**Discussion by david-botelho-mariano** [projectdiscovery/nuclei#4985] [Releases]
Good afternoon, I would like to share the recent exploit I created and would like it to be incorporated into this repository: https://github.com/david-botelho-mariano/exploit-CVE-2024-25723

**Comment by GeorginaReeder** [comment]
Hi @david-botelho-mariano ! Thanks for this. Could you please go ahead and create a pull request for this? We can then take a closer look from there.

**Reply by david-botelho-mariano** [reply]
Thanks for the guide, if you could review it and help me with the next steps, I would really appreciate it: https://github.com/projectdiscovery/nuclei-templates/issues/9513

**Comment by GeorginaReeder** [comment]
No worries, I can see we're taking a look into this for you in the [other thread](https://github.com/projectdiscovery/nuclei-templates/issues/9513) :)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
